src/shared/components/main-gradient/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { ColorScheme as ThemeColor, mainStore } from '@screens/main/store/store'
+import { LinearGradient } from 'expo-linear-gradient'
+import { type FC } from 'react'
+import { View, useWindowDimensions } from 'react-native'
+
+const MainGradient: FC = () => {
+  const { height } = useWindowDimensions()
```

Comment:
You will be better off using React native dimension
```suggestion
import { Dimensions } from 'react-native'

const { height } = Dimensions.get('window')
const MainGradient: FC = () => {
```